config/samples/flows_v1alpha1_flowcollector_versioned.yaml

```diff
@@ -4,21 +4,22 @@ metadata:
   name: cluster
 spec:
   namespace: "network-observability"
-  agent: ipfix
-  ipfix:
-    cacheActiveTimeout: 60s
-    cacheMaxFlows: 100
-    sampling: 400
-  ebpf:
-    image: 'quay.io/netobserv/netobserv-ebpf-agent:v0.1.1'
-    imagePullPolicy: IfNotPresent
-    sampling: 0
-    cacheActiveTimeout: 5s
-    cacheMaxFlows: 1000
-    interfaces: []
-    excludeInterfaces: ["lo"]
-    logLevel: info
-    privileged: false
+  agent:
+    type: IPFIX
+    ipfix:
+      cacheActiveTimeout: 60s
+      cacheMaxFlows: 100
+      sampling: 400
+    ebpf:
+      image: 'quay.io/netobserv/netobserv-ebpf-agent:v0.1.3'
```

Comment:
Are we checking the backward compatibility with this tagged image?
I just tested and I had errors, FLP being unable to connect to loki.

Reply:
Yeah we check backwards-compatibility but the image tag is wrong and we'll fix it.

Anyway, an error in the agent should not affect FLP connectivity.